tests/output/ConsoleInteractionTest.py

```diff
@@ -712,9 +712,7 @@ def test_show_bear_with_content(self):
                              '  Needed Settings:\n'
                              '   * setting1: Required Setting.\n\n'
                              '  Optional Settings:\n'
-                             '   * setting2: Optional Setting. ('
-                             "Optional, defaults to 'None'."
-                             ')\n\n'
+                             '   * setting2: Optional Setting.\n\n'
```

Comment:
@adtac What you want was already there... :worried: 
optional_params[arg] = (
                    doc_comment.param_dict.get(arg, cls.str_nodesc) + ' (' +
                    cls.str_optional.format(
                        defaults[i-num_non_defaults]) + ')',
                    argspec.annotations.get(arg, None),
                    defaults[i-num_non_defaults])

The value in tuple is ```setting description``` , second is like ```<class 'str'>``` and the third value is ```default value```

If you want that UI should remain same then the reason that we can-not remove that concatenation where we are saying ```"Optional, defaults to something``` is that we are printing this exact string using ```show_enumeration``` ```in ConsoleInteraction.py``` where we are just printing this by using ```value[0]``` for each setting and since here we concatenated this in the first value of tuple, that is why that ```Optional, defaults to something``` is coming. If we remove that concatenation then UI will change... But what you want was already there before I made changes...